### PR TITLE
feat: add crosslink banner to carbon platform on homepage

### LINKED ISSUE #3179: Develop Next release banner
Related design issue: https://github.com/carbon-design-system/carbon-website/issues/3146

For the v1 Next release, we need a banner on the homepage that will link the main Carbon site out to the new Next site. 

Interactions:
- Banner **only** exists on the homepage
- Banner is **not** sticky and scrolls away

## Spec
[Figma file](https://www.figma.com/file/H3kWgBYKdMzov7LIZ1BlHa/Next-crosslinks?node-id=67%3A6733)

Group 1

### PR DESCRIPTION
**Pull request by alisonjoseph**
Closes #3179

Add fixed top banner to homepage to crosslink to Carbon Platform.

Related Platform PR https://github.com/carbon-design-system/carbon-platform/pull/1373

#### Changelog

**New**

- Banner component

### CODE REVIEW

**Comment by aagonzales** [comment]
@alisonjoseph the preview isn't showing.

**Comment by alisonjoseph** [comment]
@aagonzales I think its still building, I'll come back and check in a bit

**Comment by alisonjoseph** [comment]
@aagonzales should be working now, https://carbondesignsystem-git-fork-alisonj-33e125-carbon-design-system.vercel.app/

File: src/components/Banner/Banner.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+const Banner = () => {
+  return (
+    <div aria-label="banner" className={banner}>
+      <p className={bannerText}>
+        <strong className={bannerHeading}>
+          A new web experience is coming!
+        </strong>
+        <span className={bannerDetails}>
+          The v1 Carbon platform v1 is now available in public beta.
```
Comment (tay1orjones) [inline src/components/Banner/Banner.js:21]: Duplicate "v1", and should platform be capitalized too?

```suggestion
          The Carbon Platform v1 is now available in public beta.
```

**Review by tay1orjones — APPROVED** [review]
One small change, but otherwise looks good

File: src/components/Banner/Banner.js (hunk 1 above)
Comment (alisonjoseph) [reply src/components/Banner/Banner.js:21]: Good catch, @aagonzales can you confirm correct copy here, the figma file has several versions depending on the breakpoint. I think it should probably be `The v1 Carbon platform is now available in public beta.` but wanted to double check. 
Screen Shot 2022-10-05 at 8 56 22 AM

**Review by tw15egan — APPROVED** [review]
LGTM once Taylor's suggestion is made 👍🏻

File: src/components/Banner/Banner.js (hunk 1 above)
Comment (aagonzales) [reply src/components/Banner/Banner.js:21]: oh sorry for the confusion, I should have updated the image in the issue. This is the final copy with suggestions from Aubrey. 

> A new web experience is coming! The v1 Carbon platform is now available in public beta.
> CTA: Try the Carbon Platform

Comment (aagonzales) [inline src/components/Banner/Banner.js:21]:
```suggestion
          The v1 Carbon platform is now available in public beta.
```

**Review by aagonzales — APPROVED** [review]
Looks good after that one content update

**Pull request merged**